core/src/main/scala/kafka/server/ReplicaAlterLogDirsThread.scala

```diff
@@ -34,7 +35,7 @@ import org.apache.kafka.common.requests.FetchResponse.PartitionData
 import org.apache.kafka.common.requests.{EpochEndOffset, FetchRequest, FetchResponse}
 
 import scala.collection.JavaConverters._
-import scala.collection.{Map, Seq, Set, mutable}
+import scala.collection.{mutable, Map, Seq, Set}
```

Comment:
It seems that Seq is unused now.

Reply:
I believe it's still used here https://github.com/apache/kafka/pull/7443/files/4a306ad7967e20712ec2c0ea085a91b5f471b9a3#diff-6446cfed30fa23cfc249b9537fffbe35R79. It's not showing up for me as a checkstyle failure.